### DISCUSSION TITLE
NEAR TodoList App

### DISCUSSION THREAD

**Discussion by kasodon** [near/near-sdk-js#328] [Show and tell]
A TodoList app built using create-react-app and near-sdk-js. Here's a [demo link](http://near-todo-app.vercel.app/). Github repo [link](https://github.com/kasodon/near-todo-app)

**Comment by ailisp** [comment]
Just tried out, intuitive user experience and beautiful UI! great work

**Reply by kasodon** [reply]
Thanks @ailisp 😊
